libnetwork/netlabel/labels.go

```diff
@@ -103,21 +97,3 @@ func MakeKVProviderConfig(scope string) string {
 func MakeKVClient(scope string) string {
```

Comment:
FWIW; this is the only function currently used. I want to dig a bit further, but from the looks of it, we only support 3 scopes;

- `swarm`
- `global`
- `local`

So perhaps we should define consts for those cases only, and remove some of the abstraction here (keeping that for a further exercise)